recipes/lazypredict/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "lazypredict" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4986103b7c98b0a675c7a8f54681435e6fefafb2041759ad8c7f4f403171cb04
+
+build:
+  number: 0
+  entry_points:
+    - lazypredict=lazypredict.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click >=7.0
```

Comment:
I believe you can remove `click` from the `host` section. You will need to add `pytest-runner`.